doc/Language/operators.pod6

```diff
@@ -1889,7 +1889,7 @@ is smaller than or equal to the second.
 
 Numeric less than or equal to operator.
 
-Equivalent to L«<=|routine/<=», at codepoint U+2264 (LESS-THAN OR EQUAL TO).
+Equivalent to L«<=/routine/<=», at codepoint U+2264 (LESS-THAN OR EQUAL TO).
```

Comment:
I don't think this this should be changed actually

Reply:
Why? The first one is not working for me.

Reply:
But you're changing a text-url separator (`|`) to a slash for directory separation. That one is not going to work no matter what. Maybe you should leave `|/` instead?

Reply:
Oh, that's right. My bad. Changed now